buildkite/src/Jobs/Release/MinaArtifactNobleMesaDevnet.dhall

```diff
@@ -23,15 +23,8 @@ in  Pipeline.build
             , Artifacts.Type.ZkappTestTransaction
             , Artifacts.Type.CreateLegacyGenesis
             ]
-          , network = Network.Type.Devnet
-          , tags =
-            [ PipelineTag.Type.Long
-            , PipelineTag.Type.Release
-            , PipelineTag.Type.Docker
-            , PipelineTag.Type.Devnet
-            , PipelineTag.Type.Amd64
-            , PipelineTag.Type.Noble
-            ]
+          , network = Network.Type.Mesa
+          , tags = [] : List PipelineTag.Type
```

Comment:
why cleaned all tags here and couple of other places?

Reply:
to disable such jobs.